### PR TITLE
Leighton Fritze: SPRINT -- User Interface & Git

### PR DESCRIPTION
**Pull request by Lfritze**

### CODE REVIEW

**Comment by Lfritze** [comment]
I discovered :nth-child here for coloring the boxes:
https://css-tricks.com/almanac/selectors/n/nth-child/

File: index.html [hunk 1]
```diff
@@ -47,29 +68,35 @@ <h2>Why Did It Have To Be Boxes...</h2>
...
+
             <div class="text-container">
                 <h2>Moon</h2>
-                <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
+                    <p>Phasellus id dui id tortor tincidunt efficitur. Proin sed quam sed tellus vestibulum ultrices quis in nunc.</p>
             </div>
 
         </section>
         
         <footer>
```
Comment (MrT3313) [inline index.html:91]: How would you change your CSS to align your footer to the left vs center

File: about.html [hunk 2]
```diff
@@ -7,67 +7,95 @@
...
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          <h2>
+            About
+          </h2>
+
+            <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+
+      </section>
+
+      <section class="middle">
```
Comment (MrT3313) [inline about.html:46]: what properties would you need to add to make the concent appear to be 'pinched in' from the edges of the container?

File: about.html [hunk 3]
```diff
@@ -7,67 +7,95 @@
...
+      <div class="boxing">
+        <div class="collaborate-img"><img class="people-pic"src="https://github.com/Lfritze/Sprint-Challenge--User-Interface/blob/master/img/about-meeting.png?raw=true" alt="strategy">
+        </div>
+          <h2>Collaboration</h2>    
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+              <a href="#">Learn More</a>
+      </div><!-- closes boxing div -->
+    </section> <!-- closes middle section -->
+      
+    <section class="bottom-part">
```
Comment (MrT3313) [inline about.html:81]: what properties would you need to add to make the content appear to be 'pinched in' from the edges of the container?

**Review by MrT3313 — COMMENTED** [review]
- please change the name of the PR to: Leighton Fritze: SPRINT -- User Interface & Git
- Dont forget your read me answers! 
- try to make your <opening> </closing> tags are on the same indentation level

**Pull request merged**